tests/test_mpodApalis.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import pytest
+from ophyd.sim import make_fake_device
+from pcdsdevices.device_types import MPODApalisModule4Channel
+from pcdsdevices.mpodApalis import MPODApalisChannel, MPODApalisCrate
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def fake_mpod_channel1():
+    FakeMPODChannel = make_fake_device(MPODApalisChannel)
+    C1 = FakeMPODChannel('TEST:MPOD:CHANNEL1', name='TestC1')
+    # switch off
+    C1.state.sim_put(0)
+    C1.voltage.sim_put(50)
+    C1.current.sim_put(0.02)
+    C1.max_voltage.sim_put(100)
+    C1.max_current.sim_put(0.1)
+    return C1
+
+
+@pytest.fixture(scope='function')
+def fake_mpod_channel2():
+    FakeMPODChannel = make_fake_device(MPODApalisChannel)
+    C2 = FakeMPODChannel('TEST:MPOD:CHANNEL2', name='TestC2')
+    # switch off
+    C2.state.sim_put(0)
+    C2.voltage.sim_put(90)
+    C2.current.sim_put(0.01)
+    C2.max_voltage.sim_put(100)
+    C2.max_current.sim_put(0.1)
+    return C2
+
+
+@pytest.fixture(scope='function')
+def fake_mpod_module4Channel():
+    FakeMPODmodule = make_fake_device(MPODApalisModule4Channel)
+    mpod_module4 = FakeMPODmodule('TEST:MPOD:MOD:4', name='TestM4')
+    mpod_module4.voltage_ramp_speed.sim_put(10)
+    mpod_module4.current_ramp_speed.sim_put(10)
+    mpod_module4.faults.sim_put(0)
+    return mpod_module4
+
+
+@pytest.fixture(scope='function')
+def fake_mpod_crate():
+    FakeMPODCrate = make_fake_device(MPODApalisCrate)
+    mpod = FakeMPODCrate('TEST:MPOD', name='test')
+    # switch on
+    mpod.power.sim_put(1)
+    return mpod
+
+
+def test_switch_on_off(fake_mpod_channel1):
+    logger.debug('Testing MPOD Channel On/Off state')
+    assert fake_mpod_channel1.state.get() == '0'
+    fake_mpod_channel1.on()
+    assert fake_mpod_channel1.state.get() == '1'
```

Comment:
A note to other reviewers: this is a string because the signal is set to `string=True` but no enum strings are defined for the fake channels. So this means instead of looking up the str value of 1, it just casts the integers as strings instead. This will show up as a readable text string when using the real device I think.